### DISCUSSION TITLE
Publish Unstable Release to Github Packages

### DISCUSSION THREAD

**Discussion by septs** [OpenINF/.github#26] [Ideas]
About no-authorization access github packages (npm registry) problem:

see https://github.com/NiceLabs/npm-registry-worker

**Comment by septs** [comment]
usage e.g:

```
npm install @openinf/eslint-plugin --registry=https://npm.open.inf.is
```

**Comment by DerekNonGeneric** [comment]
Oh, this is great to be able to do! That way we don't end up putting junk on the registry that we can't remove later, right?
